### PR TITLE
Feat request class

### PR DESCRIPTION
**Pull request by hcfw007**
WIP

### CODE REVIEW

**Comment by hcfw007** [comment]
Related issue: [封装 request 类，处理所有api请求](#21)

**Comment by hcfw007** [comment]
ping @su-chang

File: src/work/manager.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+export class Manager {
+
+  client: Client
+  requestManager: RequestManager
+
+  constructor () {
+
+    this.client = new Client({})
+    void this.client.initialize()
```
Comment (su-chang) [inline src/work/manager.ts:12]: Which class created the `Manager` class?

Comment (hcfw007) [reply src/work/manager.ts:12]: puppetWhatsapp. I thought you did that.

**Comment by hcfw007** [comment]
Still need to move most of operations from puppet-whatsapp to manager

**Review by su-chang — APPROVED** [review]

**Pull request merged**